lib/money_column/active_record_hooks.rb

```diff
@@ -1,41 +0,0 @@
-module MoneyColumn
-  module ActiveRecordHooks
-    def self.included(base)
-      base.extend(ClassMethods)
-    end
-
-    module ClassMethods
-      def money_column(*columns, currency_column: 'currency')
-        Array(columns).flatten.each do |name|
-          define_method(name) do
-            value = read_attribute(name)
-            return nil if value.blank?
-            currency = read_attribute(currency_column)
-            Money.new(value, currency)
-          end
-
-          define_method("#{name}_before_type_cast") do
-            send(name) && sprintf("%.2f", send(name))
-          end
-
-          define_method("#{name}=") do |value|
-            if value.blank? || !value.respond_to?(:to_money)
-              write_attribute(name, nil)
-              nil
-            else
-              money = value.to_money
```

Comment:
I've updated the PR description with links to AR docs, the attributes API example has shows how the value can be normalized at the ORM layer. Or you do it at the controller layer (call `Money.parse` yourself or something like stronger_parameters). Either way there's no reason to keep this around.